### PR TITLE
Fix fmt formatings strings

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: include/veriblock/pop/entities/endorsement.hpp [hunk 1]
```diff
@@ -98,7 +98,7 @@ struct Endorsement {
 
   std::string toPrettyString(size_t level = 0) const {
     return format(
-        "{}{}Endorsement{id={}, containing={}, endorsed={}, blockOfProof={}}",
+        "{}{}Endorsement{{{id={}, containing={}, endorsed={}, blockOfProof={}}}}",
```
Comment (blackyblack) [inline include/veriblock/pop/entities/endorsement.hpp:101]: Remove extra open and closing brace. Should be Endorsement{{

File: src/pop/entities/btcblock.cpp [hunk 2]
```diff
@@ -57,8 +57,8 @@ const BtcBlock::hash_t& BtcBlock::getHash() const {
 
 std::string BtcBlock::toPrettyString() const {
   return format(
-      "BtcBlock{version=%{}, prev={}, merkleRoot={}, timestamp=%{}, "
-      "bits={}, nonce={}}",
+      "BtcBlock{{version=%{}, prev={}, merkleRoot={}, timestamp=%{}, "
+      "bits={}, nonce={}}}",
```
Comment (blackyblack) [inline src/pop/entities/btcblock.cpp:61]: Remove % symbols. Eg version={}

**Review by blackyblack — CHANGES_REQUESTED** [review]

File: include/veriblock/pop/entities/endorsement.hpp (hunk 1 above)
Comment (Mr-Leshiy) [reply include/veriblock/pop/entities/endorsement.hpp]: fixed

File: src/pop/entities/btcblock.cpp (hunk 2 above)
Comment (Mr-Leshiy) [reply src/pop/entities/btcblock.cpp:61]: fixed

**Review by blackyblack — APPROVED** [review]

**Pull request merged**